spec/controllers/logix_controller_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe LogixController, type: :request do
+  it 'should get index page' do
+    get root_path
+    expect(response.status).to eq(200)
+  end
+  it 'should get gettingStarted' do
```

Comment:
Empty line before each it block, just like methods

Reply:
Ok,got it.